ssp

```diff
@@ -1755,20 +1762,7 @@ sub check_for_cpanel_update {
     my $local_tier_name;
     my $match = 0;
 
-    my $cpupdate_conf = '/etc/cpupdate.conf';
-    return if !-f $cpupdate_conf;
-
-    if ( open my $file_fh, '<', $cpupdate_conf ) {
-        while (<$file_fh>) {
-            if (/\bcpanel=(.*)/i) {
-                $local_tier_name = $1;
-                last;
-            }
-        }
-        close $file_fh;
-    }
-
-    return if !$local_tier_name;
+    return unless defined $CPUPDATECONF->{CPANEL};
```

Comment:
Isn't $CPUPDATECONF->{CPANEL}; always going to be defined here? On line 1720 tht ternary is going to populate that scalar no matter what - just one way with the proper version and the other with a string representing an error state.  Would it be better to just populate the var on 1720 and move a conditional return here?

Reply:
The ternary on 1720 won't define $CPUPDATECONF->{CPANEL}, it tests its defined-ness before using it, and does something else if it isn't.  1765 lies inside of a completely different function, so its defined-ness should be tested again.